tools/rdiff-backup.spec.template

```diff
@@ -2,18 +2,22 @@
 %define PYTHON_VERSION %(%{PYTHON_NAME} -c 'import sys; print(sys.version[:3])')
 %define NEXT_PYTHON_VERSION %(%{PYTHON_NAME} -c 'import sys; print("%d.%d" % (sys.version_info[0], sys.version_info[1]+1))')
 
+# Note this in not necessarily the same at the version, so confirm
+%define gittag %{version}
```

Comment:
Ah, good point, for the first beta version it isn't, but we can and should make sure that it is the same

Reply:
Thanks.